.github/workflows/ci.yml

```diff
@@ -0,0 +1,174 @@
+        with:
+          name: packages
+          path: |
+            packages/**/dist
+
+  deploy_openconceptlab_app:
+    runs-on: ubuntu-latest
+
+    env:
+      DIR_NAME: "esm-admin-openconceptlab-app"
+      ESM_NAME: "@openmrs/esm-openconceptlab-app"
+      JS_NAME: "openmrs-esm-openconceptlab-app.js"
+
+    needs: build
+
+    if: ${{ github.event_name == 'push' }}
+
+    steps:
+      - name: Download Artifacts
+        uses: actions/download-artifact@v2
+      - name: Compute Timestamp
+        run: echo "TIMESTAMP=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+      - name: Prepare Directory
+        shell: bash
+        run: |
+          mkdir -p dist/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}
+          mv packages/${{ env.DIR_NAME }}/dist/*.* dist/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}/
+      - name: Publish to Digital Ocean
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --cache-control "max-age=31536000"
+        env:
+          AWS_S3_BUCKET: ${{ secrets.DIGITAL_OCEAN_SPACES_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.DIGITAL_OCEAN_SPACES_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.DIGITAL_OCEAN_SPACES_ACCESS_KEY }}
+          AWS_S3_ENDPOINT: ${{ secrets.DIGITAL_OCEAN_SPACES_ENDPOINT }}
+          SOURCE_DIR: "dist"
+      - name: Update Importmap
+        uses: fjogeleit/http-request-action@master
+        with:
+          url: http://${{ secrets.DEPLOYER_HOST }}/services?env=prod
+          method: "PATCH"
+          username: ${{ secrets.DEPLOYER_USERNAME }}
+          password: ${{ secrets.DEPLOYER_PASSWORD }}
+          data: '{ "service":"${{ env.ESM_NAME }}","url":"https://spa-modules.nyc3.cdn.digitaloceanspaces.com/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}/${{ env.JS_NAME }}" }'
+          customHeaders: '{ "Accept": "application/json", "Content-Type": "application/json" }'
+
+  pre_release:
+    runs-on: ubuntu-latest
+
+    needs: build
+
+    if: ${{ github.event_name == 'push' }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "17"
+          registry-url: "https://registry.npmjs.org"
+      
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npx lerna bootstrap
+      
+      - name: Build
+        run: yarn turbo run build --color --concurrency=5
+
+      - name: Patch
+        run: yarn lerna version patch --no-git-tag-version --no-push --yes
+      
+      - name: Version
+        run: yarn lerna version "$(node -e "console.log(require('./lerna.json').version)")-pre.${{ github.run_number }}" --no-git-tag-version --yes
```

Comment:
Please collapse these into one as is done [here](https://github.com/openmrs/openmrs-esm-core/blob/276d11c16395faf91c6f3b28cef697329246d3f5/.github/workflows/ci.yml#L75). It'll be easier in the long-run.